### DISCUSSION TITLE
Cetveldeki silik yerler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#106] [Zeka Sorusu]
33 cm uzunluğunda bir cetvel var. Bu cetveldeki bazı yerler silik.
Tam olarak okunabilen 8 işaret var.
Bu 8 işaretle 1 cm’den 33 cm’ye kadar olan bütün tam sayı uzunluklar ölçülebiliyor.
Cetvel üzerinde okunabilen sayılar nelerdir?

**Comment by Eceg25** [comment]
0, 2, 5, 9, 13, 18, 25, 33

**Comment by Sevvalgocgun1** [comment]
Cetvelde 8 işaretle 1 cm'den 33 cm'ye kadar olan tüm tam sayı uzunlukları ölçebilmek için cetveldeki işaretler şu şekilde yerleştirilebilir:

1, 3, 6, 10, 15, 21, 28, 36

Bu işaretler, her bir işaretin arasındaki farkları kullanarak 1 cm’den 33 cm’ye kadar olan tüm tam sayı uzunluklarını ölçmenizi sağlar.
